src/main/java/nl/tudelft/pixelperfect/game/Settings.java

```diff
@@ -0,0 +1,50 @@
+package nl.tudelft.pixelperfect.game;
+
+/**
+ * A singleton data class for settings tracking that are used across the game.
+ * 
+ * @author David Alderliesten
+ *
+ */
+public final class Settings {
```

Comment:
Adding every setting as an attribute to this class requires us to update the class each time we introduce a new setting: it is not very flexible. I suggest providing the Settings class with a Map (or a couple of them for different data types) containing key / value pairs describing the settings. Then we only need two methods (get and set) rather than one attribute and two methods per setting.